### ISSUE TITLE
Feature: Saying Hi

### ISSUE THREAD

**Issue by mattyjones** [shanselman/NightscoutPyPortal#2]
I am also working on this based on the latest ADABOX ;) I have a similar project written in Python that also does alerts via [LIFX](https://www.lifx.com/) lights. For example critical would turn on the lights to bright red, stale data would be purple, etc.

I am thinking a little different than you as I was going to put the data in a time-series [db](https://github.com/influxdata/influxdb) and then use stock tools such as [grafana](https://grafana.com/) to build a separate dashboard for my Pi. Think of it like Nightscout light, I just want the raw data, not the UI business.

Long story short I was just stopping by to say hi and if you had thoughts or wished to work together I would be open to it. Both my young sons are T1D using Dexcom so I have been actively hacking on IoT devices and such for the several months when I have time with the goal of building a raspberry pi powered interface in python to monitor and help manage their T1D.

**Comment by shanselman** [comment]
Love it. I'm happy to help but it does seem like you have a more vested interest. Do you want to take over and move this to your world? I'll happily help, but it feels like you've done more work now than I!

**Comment by mattyjones** [comment]
Your call. I am in the process of talking with Nightscout to maybe create several hardware repos for projects like this. We can defer to my fork if you want and I will add a blurb to the readme giving you credit for the initial functionality.

**Comment by shanselman** [comment]
Sounds great! let me know how I can help!
--
Scott Hanselman
@shanselman <http://twitter.com/shanselman> - Podcast:
https://hanselminutes.com/shows
